pkg/addons/data/SecurityContextConstraints.security.openshift.io/privileged.yaml

```diff
@@ -36,7 +36,6 @@ supplementalGroups:
 users:
 - system:admin
 - system:serviceaccount:openshift-infra:build-controller
-- system:serviceaccount:openshift-azure:default
```

Comment:
@jim-minter unclear why this was needed.

Reply:
I think it is leftover from tunnel

Reply:
Ah, right, cool.